### PR TITLE
ADDING: shape2D circle and rectangle

### PR DESCRIPTION
**Pull request by mangasteak**

### CODE REVIEW

File: src/Shape2D.cpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include "Shape2D.hpp"
+#include "raylib.h"
+
+indie::Shape2D::Shape2D()
```
Comment (antoinepnt) [inline src/Shape2D.cpp:4]: Do namespace indie  {your code} instead

File: src/Shape2D.hpp [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+{
+    /**struct Shape2D;
+     * @brief Shape2D object
+     */
+    class Shape2D
+    {
+    public:
+        /**
+         * @brief Construct a new Shape2D object
+         */
```
Comment (antoinepnt) [inline src/Shape2D.hpp:17]: You can use "///" instead of "/**" when there's only one line

File: src/Shape2D.hpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        ~Shape2D();
+
+        /**
+         * @brief Draw circle
+         * @param centerX position X of the center of the circle
+         * @param centerY position Y of the center of the circle
+         * @param radius radius of the circle
+         * @param color Color of the circle
+         */
+        void drawCircle(int centerX, int centerY, float radius, Color color);
```
Comment (antoinepnt) [inline src/Shape2D.hpp:31]: Maybe add a default color to the function's prototype (WHITE for example)

**Review by antoinepnt — CHANGES_REQUESTED** [review]

**Review by antoinepnt — APPROVED** [review]

File: src/Shape2D.hpp [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        ~Shape2D();
+
+        /**
+         * @brief Draw circle
+         * @param centerX position X of the center of the circle
+         * @param centerY position Y of the center of the circle
+         * @param radius radius of the circle
+         * @param color Color of the circle
+         */
+        static void drawCircle(int centerX, int centerY, float radius, Color color = WHITE);
```
Comment (iDommel) [inline src/Shape2D.hpp:26]: Where is the WHITE color defined ? Doest this compile ?

**Review by iDommel — APPROVED** [review]

**Pull request merged**